Bylaws.md

```diff
@@ -48,34 +52,37 @@ Chartered Working Group (CWG) Members are automatically granted Forum Membership
   
- 4. Upon commencement of the voting period, Voting Members shall have exactly seven (7) calendar days for voting on the proposed ballot, with the deadline clearly communicated in the ballot and sent via a Public Mail List. For ballots related to the Forum level, votes must be sent to the Public Mail List of the Forum. For ballots related to a CWG, votes must be sent to the Public Mail List of the CWG. All voting will take place via a Public Mail List. Votes not submitted to the proper Public Mail List, or not submitted by a designated Voting Representative, by the end of the voting period (as specified in the ballot) will be considered invalid, and will not be counted for any purpose. Each Voting Member, and not the Forum or CWG, will be responsible for taking precautions to make sure such Member’s vote is submitted properly and counted. In the event that a Voting Member’s vote on a ballot is not submitted properly, such vote shall not be valid and shall not be counted for any purpose, and there shall be no appeal, revote (except in the case of a new ballot submitted to all Voting Members) or other recourse.
+4. Upon commencement of the voting period, Voting Members shall have exactly seven (7) calendar days for voting on the proposed ballot, with the deadline clearly communicated in the ballot and sent via a Public Mail List. A proposer may withdraw a ballot containing defective language at any point during the voting period. For ballots related to the Forum level, votes must be sent to the Public Mail List of the Forum. For ballots related to a CWG, votes must be sent to the Public Mail List of the CWG. All voting will take place via a Public Mail List. Votes not submitted to the proper Public Mail List, or not submitted by a Voting Representative, by the end of the voting period (as specified in the ballot) will be considered invalid, and will not be counted for any purpose. Each Voting Member, and not the Forum or CWG, will be responsible for taking precautions to make sure such Member's vote is submitted properly and counted. In the event that a Voting Member’s vote on a ballot is not submitted properly, such vote shall not be valid and shall not be counted for any purpose, and there shall be no appeal, re-vote (except in the case of a new ballot submitted to all Voting Members) or other recourse.
 
  5. Voting Members may vote “yes”, “no”, or “abstain” on a ballot. Only votes that indicate a clear “yes” or “no” response to the ballot question shall be considered (i.e. votes to “abstain” and votes that do not indicate a clear “yes” or “no” response will not figure in the calculation of item (6), below).
```

Comment:
Aside from “yes”, “no”, or “abstain” should any other commentary be included in the vote? Section 2.3(4) requires votes be "submitted properly" and I am wondering if the content of the vote needs to be specified for submission?

Reply:
Commentary has always been allowed. This "yes", "no", or "abstain" language is to indicate how the actual vote must be cast. The language for "submitted properly" was added to handle a case where a voting member submitted the vote from a representative that was not allowed to post to the public list, so that vote did not count. We've never had any issues since.

I suggest we keep this language as is, but if you want to suggest any improvements, please do.

Reply:
Thank you for the explanation. I think I was having a hard time with the use of "proper" and "properly" in 2.3(4). Here is a suggestion for consideration in that section:

From: "Votes not submitted to the proper Public Mail List, or not submitted by a Voting Representative, by the end of the voting period (as specified in the ballot) will be considered invalid, and will not be counted for any purpose."

To: "Votes must be submitted by a Voting Representative to the correct Public Mail List by the end of the voting period (as specified in the ballot) to be considered valid and counted."